.rubocop.yml

```diff
@@ -171,6 +171,9 @@ RSpec/MultipleMemoizedHelpers:
 RSpec/NestedGroups:
   Enabled: false
 
+RSpec/NamedSubject:
+  Enabled: false
```

Comment:
I guess it's to prevent yourself from being spamed while updating the spec files?

Reply:
Yeah. I could update the files I changed but there are a lot other tests using unnamed subject, which tbh I think is fine ^^

![CleanShot 2024-04-25 at 09 29 24@2x](https://github.com/getlago/lago-api/assets/1525636/1298a355-13b6-445b-96a9-66efc61d81fd)